src/main/scala/org/apache/spark/sql/delta/sources/DeltaDataSource.scala

```diff
@@ -342,4 +343,44 @@ object DeltaDataSource extends DatabricksLogging {
       None
     }
   }
+
+  /**
+   * Extends the DataFrameReader API by adding a delta function
+   * Usage: spark.read.delta(path)
```

Comment:
Why not to use something like that would get formatted properly in scala/javadoc?

```
 * {{{
 * spark.read.delta(path)
 * }}}
```

Reply:
done 👍